keyboards/keebio/quefrency/rules.mk

```diff
@@ -31,5 +31,3 @@ RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
 
 SPLIT_KEYBOARD = yes
-
-DEFAULT_FOLDER = keebio/quefrency/rev1
```

Comment:
Removing this is making travis CI fail. 

Basically this needs to be re-added, or all this stuff moved out of the quefrency folder, and into the specific revisions.

Reply:
Alright, I'll just revert for now.